### PR TITLE
Millis

### PR DESCRIPTION
**Pull request by cbolles**
Adds a function to EVT-core for getting the current system time in milliseconds. This is just a wrapper around `HAL_GetTick` for exposing a common interface.

### CODE REVIEW

**Review by zsteig — APPROVED** [review]

File: samples/millis/main.cpp [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/**
```
Comment (mjmagee991) [inline samples/millis/main.cpp:1]: This file needs an update to its documentation.

**Review by mjmagee991 — CHANGES_REQUESTED** [review]
Looks good. Just seems like you missed the documentation in the sample file.

File: samples/millis/main.cpp (hunk 1 above)
Comment (cbolles) [reply samples/millis/main.cpp:1]: Nice catch

**Review by mjmagee991 — APPROVED** [review]
Nice work

**Pull request merged**